### ISSUE TITLE
$ref '/definitions/jspmDefinition' in {1} can not be resolved.'

### ISSUE THREAD

**Issue by ProtonScott** [Microsoft/vscode#33222]
- VSCode Version: Code 1.15.1 (41abd21afdf7424c89319ee7cb0445cc6f376959, 2017-08-16T17:15:57.756Z)
- OS Version: Darwin x64 16.6.0
- Extensions: 
AESSoft.aessoft-class-autocomplete
AndersEAndersen.html-class-suggestions
CoenraadS.bracket-pair-colorizer
DanTup.dart-code
DotJoshJohnson.xml
Gigabyte-Giant.vscode-file-header-comment-helper
HookyQR.beautify
HookyQR.CreateModule
HookyQR.minify
IBM.output-colorizer
MS-DevSkim.vscode-devskim
Shan.code-settings-sync
SirTori.indenticator
Zignd.html-css-class-completion
abierbaum.vscode-file-peek
akamud.vscode-caniuse
ashhitch.vs-code-css-comments
bajdzis.vscode-database
bibhasdn.git-easy
bierner.color-info
bmewburn.vscode-intelephense-client
christian-kohler.npm-intellisense
christian-kohler.path-intellisense
cmstead.jsrefactor
cssho.vscode-svgviewer
dbaeumer.jshint
dbaeumer.vscode-eslint
deerawan.vscode-dash
donjayamanne.githistory
donjayamanne.jquerysnippets
donjayamanne.python
eamodio.gitlens
ecmel.vscode-html-css
emilast.LogFileHighlighter
emmanuelbeziat.vscode-great-icons
eriklynd.json-tools
esbenp.prettier-vscode
felipecaputo.git-project-manager
felixfbecker.php-debug
felixfbecker.php-intellisense
felixfbecker.php-pack
formulahendry.auto-close-tag
formulahendry.auto-rename-tag
formulahendry.code-runner
hackwaly.ocaml
hbenl.vscode-firefox-debug
howardzuo.vscode-git-tags
huizhou.githd
humao.rest-client
ionutvmi.path-autocomplete
joshpeng.sublime-babel-vscode
joshpeng.theme-onedark-sublime
justusadam.language-haskell
kamikillerto.vscode-colorize
karyfoundation.theme-karyfoundation-themes
lamartire.git-indicators
leizongmin.node-module-intellisense
lihui.vs-color-picker
magicstack.MagicPython
mdickin.markdown-shortcuts
michelemelluso.code-beautifier
mkaufman.HTMLHint
mrmlnc.vscode-apache
mrmlnc.vscode-attrs-sorter
mrmlnc.vscode-autoprefixer
mrmlnc.vscode-csscomb
mrmlnc.vscode-doiuse
mrmlnc.vscode-duplicate
mrmlnc.vscode-json5
mrmlnc.vscode-postcss-sorting
mrmlnc.vscode-remark
mrmlnc.vscode-scss
mrmlnc.vscode-stylefmt
msjsdiag.debugger-for-chrome
msjsdiag.debugger-for-ios-web
naumovs.color-highlight
plievone.vscode-template-literal-editor
pranaygp.vscode-css-peek
rebornix.Ruby
robertohuertasm.vscode-icons
sbrink.elm
shaharkazaz.git-merger
shakram02.bash-beautify
shyykoserhiy.git-autoconfig
spywhere.guides
steoates.autoimport
timonwong.shellcheck
vikas.code-navigation
vitorsalgado.vscode-redis
waderyan.gitblame
wayou.vscode-todo-highlight
webfreak.code-d
wholroyd.jinja
wix.vscode-import-cost
zhuangtongfa.Material-theme
---

Steps to Reproduce:

1. I'm unsure when this appeared, but I had installed Eslint extension and went to put the ESLint settings in my json.settings
2. At which point I noticed, when going back and forth testing my eslint configuration, this:

```
file: 'vscode://defaultsettings/settings.json'
severity: 'Warning'
message: '$ref '/definitions/jspmDefinition' in {1} can not be resolved.'
at: '1,1'
source: ''
```

I'm myself unsure whats happening here. When I load with no extensions, yes, it does not reproduce, but adding the extensions one by one doesn't seem to reproduce either. From what I can tell, when I actually double click on that error, it takes me to the settings.json  location in vscode:// and at the top and bottom of the file, above the curly braces, there is a opening bracket '[' and closing bracket ']' respectively. I imagine that’s whats throwing that error. I have attempted to try and just edit that file, and it appears I can't. I can't seem to find it either (For the record, I'm on a Mac, 10.12.6)

Reproduces without extensions: No

**Comment by hoovercj** [comment]
Is this a duplicate of https://github.com/Microsoft/vscode/issues/31932?

@aeschli - when should that fix be in vs code? Is it already live in insiders? Im not sure how it works when the fix is in a different repo like this fix was

**Comment by aeschli** [comment]
yes dup of #31932. The fix is in 1.16.
